src/idl/wholeBodyDynamics_IDLServer/wholeBodyDynamics_IDLServer.thrift

```diff
@@ -80,6 +80,13 @@ service wholeBodyDynamics_IDLServer
   */
   bool resetOffset(1:string calib_code)
 
+  /**
+   * Use the offline estimated offset of the sensor.
+   * @param calib_code argument to specify the sensors to reset (all,arms,legs,feet)
+   * @return true/false on success/failure
+   */
```

Comment:
Probably it could make sense to mention here that for sensor that do not have a pre-estimated offset, the offset is not change at all.

Reply:
Done